docs/blog/2024-09-20-nx-19-8.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Nx 19.8 Update!!
+slug: nx-19-8-update
+authors: [Zack DeRose]
+tags: [nx, release]
+cover_image: /blog/images/2024-09-20/thumbnail.png
+---
+
+Nx 19.8 is here! This is our last minor release before we get ready to move ahead into Nx v20, which should land in October around the same time as the [Monorepo World Conference]()!
+
+As always, you can find the general details for all Nx releases on our [changelog](/changelog), as well as details on our [Github Releases for the Nx repo](https://github.com/nrwl/nx/releases).
+
+## Table of Contents
+
+In this blog post:
+
+- [Nx Import](#nx-import)
+- [Project Crystal Comes to Angular](#project-crystal-comes-to-angular)
+- [New Nx Workspaces Create with ESLint v9](#new-nx-workspaces-created-with-eslint-v9)
+- [Nx Release Enhancements](#nx-release-enhancements)
+- [Migrate to Latest](#migrate-to-latest)
+- [Round 2 of Monorepo World Conference Speakers Announced!!](#round-2-of-monorepo-world-conference-speakers-announced)
+- [Learn More](#learn-more)
+
+## Nx Import
+
+In Nx 19.8, [`nx import`](/nx-api/nx/documents/import) has now moved from beta support to now generally available!
+
+Nx Import is a new [top-level command of the Nx CLI](/reference/nx-commands) which allows you to import projects along with its git history from some other repository into your current Nx worksapce.
+
+Keep an eye out for more on Nx Import on our [YouTube Channel](https://www.youtube.com/@nxdevtools) coming soon, and in the meantime be sure to check [the documentation](/nx-api/nx/documents/import) as this is now fully documented!
+
+## Project Crystal Comes to Angular
+
+[Project Crystal](/concepts/inferred-tasks) has come to Angular!
+
+Project Crystal allows Nx to _infer_ tasks for projects in your workspace - rather than requiring that they exist in every `project.json` or `angular.json` file of your workspace.
+
+## New Nx Workspaces Created with ESLint v9
+
+When creating a new workspace with the command: `npx create-nx-workspace`, those workspaces will now be created with [`eslint`](https://www.npmjs.com/package/eslint) v9, and [`typescript-eslint`](https://www.npmjs.com/package/typescript-eslint) v8 - their most recent versions respectively.
+
+## Nx Release Enhancements
```

Comment:
Also mention this feature from a Nx Champion, Jonathan Gelin.

Ability to use `groupPreVersionCommand` in addition to `preVersionCommand` when using release groups (https://nx.dev/recipes/nx-release/build-before-versioning#build-before-versioning).